source/common/network/utility.cc

```diff
@@ -700,8 +706,10 @@ Api::IoErrorPtr Utility::readPacketsFromSocket(IoHandle& handle,
               : (packets_dropped + (std::numeric_limits<uint32_t>::max() - old_packets_dropped) +
                  1);
       // TODO(danzh) add stats for this.
-      ENVOY_LOG_MISC(
-          debug, "Kernel dropped {} more packets. Consider increase receive buffer size.", delta);
+      ENVOY_LOG_MISC(debug,
```

Comment:
can we add a stat with this PR?

Reply:
My preference would be to not add more to this PR given this already existed. I promise I will follow up with a dedicated PR to add this stat?